src/main/java/org/synyx/urlaubsverwaltung/application/statistics/ApplicationForLeaveStatisticsCsvExportService.java

```diff
@@ -71,7 +72,7 @@ void writeStatistics(FilterPeriod period, List<ApplicationForLeaveStatistics> st
             csvRow[3] = decimalFormat.format(applicationForLeaveStatistics.getTotalAllowedVacationDays());
             csvRow[4] = decimalFormat.format(applicationForLeaveStatistics.getTotalWaitingVacationDays());
             csvRow[5] = decimalFormat.format(applicationForLeaveStatistics.getLeftVacationDays());
-            csvRow[6] = decimalFormat.format(applicationForLeaveStatistics.getLeftOvertime());
+            csvRow[6] = decimalFormat.format(BigDecimal.valueOf((double) applicationForLeaveStatistics.getLeftOvertime().toMinutes() / 60));
```

Comment:
Könnten wir es hier nicht auch als stunden und Minuten rausgeben? Wie wird in excel gerne gerechnet?

Reply:
Denke, hier ist die Dezimaldarstellung das richtige um damit "Rechnen" zu können. Das ist glaube ich auch das gängige Verhalten, wenn man zwei Datumsfelder voneinander abzieht.